### PR TITLE
Qdk may rel notes

### PR DESCRIPTION
**Pull request by bromeg**
Relnotes for May release

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/c86c06033a99502b5cb273c63bfe029505255d2e/articles/relnotes/index.md) | 76 | 40 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c569881-b858-4a6f-b2d6-690021ac427b_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: June 3rd, 2020*
+
+This release contains the following:
+
+- IQ# will no longer fail when a Q# entry point is present
+- Updates to [Standard library](xref:microsoft.quantum.libraries.standard.intro) to use access modifiers
+- Compiler now allows plug-in of rewrite steps between built-in rewrite steps, see [here]() for more info
```
Comment (cgranade) [inline articles/relnotes/index.md:27]:
```suggestion
- Compiler now allows plug-in of rewrite steps between built-in rewrite steps, see [here]() for more info
- Several deprecated functions and operations have been removed following the schedule described in our [API principles](xref:microsoft.quantum.contributing.api-design). Q# programs and libraries that build without warnings in version 0.11.2004.2825 will continue to work unmodified.
```

Comment (bettinaheim) [inline articles/relnotes/index.md:27]:
```suggestion
- Compiler now allows plug-in of rewrite steps between built-in rewrite steps
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8b8221cde11cce882bbb9a0992f8c4e9f5838a85/articles/relnotes/index.md) | 76 | 40 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4f069ee-0e7d-4aef-8faa-5e668b1775e8_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8b8221cde11cce882bbb9a0992f8c4e9f5838a85/articles/relnotes/index.md) | 76 | 40 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e1ca59b-aaa4-4781-b8b8-e43bac6b9664_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8b8221cde11cce882bbb9a0992f8c4e9f5838a85/articles/relnotes/index.md) | 76 | 40 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/162c3ad0-3fbc-48af-91a7-1a6e3744e698_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 2]
```diff
@@ -16,6 +16,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+*Release date: June 3rd, 2020*
+
+This release contains the following:
+
+- IQ# will no longer fail when a Q# entry point is present
+- Updates to [Standard library](xref:microsoft.quantum.libraries.standard.intro) to use access modifiers
+- Compiler now allows plug-in of rewrite steps between built-in rewrite steps
+- Several deprecated functions and operations have been removed following the schedule described in our [API principles](xref:microsoft.quantum.contributing.api-design). Q# programs and libraries that build without warnings in version 0.11.2004.2825 will continue to work unmodified.
+
+See the full list of closed PRs for [libraries](https://github.com/Microsoft/QuantumLibraries/pulls?q=is%3Apr+is%3Aclosed), [compiler](https://github.com/microsoft/qsharp-compiler/pulls?q=is%3Apr+is%3Aclosed), [runtime](https://github.com/microsoft/qsharp-runtime/pulls?q=is%3Apr+is%3Aclosed), [samples](https://github.com/Microsoft/Quantum/pulls?q=is%3Apr+is%3Aclosed) and [Katas](https://github.com/microsoft/QuantumKatas/pulls?q=is%3Apr+is%3Aclosed).
```
Comment (rmshaffer) [inline articles/relnotes/index.md:30]: Do we want to also include the iqsharp repo in this list? I see we haven't in the past; not sure if there was a reason for that or if it's just an omission.

File: articles/relnotes/index.md [hunk 3]
```diff
@@ -16,6 +16,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: June 3rd, 2020*
+
+This release contains the following:
+
+- IQ# will no longer fail when a Q# entry point is present
```
Comment (rmshaffer) [inline articles/relnotes/index.md:25]:
```suggestion
- Q# notebooks and Python host programs will no longer fail when a Q# entry point is present
```

File: articles/relnotes/index.md (hunk 2 above)
Comment (bromeg) [reply articles/relnotes/index.md:30]: oh wow, we definitely should. I think I always just copy/pasted what was there before.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/00f45089e655930a06684a79b650d934ac82d2d8/articles/relnotes/index.md) | 76 | 40 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5aebff73-287b-4ece-8ec7-0f13f8f2e5be_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2ee2457b85e53d8034bfbace77351df5d753706a/articles/relnotes/index.md) | 76 | 41 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94a536d8-a5c8-4338-943a-77bb440812c4_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5cb244f266abb7596362b9cac5418d83f69c1e0f/articles/relnotes/index.md) | 76 | 41 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d60af964-787b-4fc9-ab79-c50a54401c40_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**